lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -491,7 +491,8 @@ isSubrangeOf r1 r2 =
                                   Stake Pools
 -------------------------------------------------------------------------------}
 
--- | Represents a stake pool.
+-- | Identifies a stake pool. A 'PoolId' is the blake2b-256 hash of the stake
+-- pool registration certificate.
```

Comment:
**[remark]**

Yes and no. This is true for Jörmungandr, but not generally true of PoolIds. We have to be careful about these things.

Reply:
OK